psi4/src/read_options.cc

```diff
@@ -1937,11 +1939,11 @@ int read_options(const std::string &name, Options & options, bool suppress_print
     options.add_int("CACHELEVEL",2);
     /*- Specifies the choice of representation of the electric dipole operator.
     Acceptable values are ``LENGTH`` for the usual length-gauge representation,
-    ``VELOCITY`` for the modified velocity-gauge representation in which the
+    ``VELOCITY``(default) for the modified velocity-gauge representation in which the
     static-limit optical rotation tensor is subtracted from the frequency-
     dependent tensor, or ``BOTH``. Note that, for optical rotation calculations,
     only the choices of ``VELOCITY`` or ``BOTH`` will yield origin-independent results. -*/
-    options.add_str("GAUGE","LENGTH", "LENGTH VELOCITY BOTH");
+    options.add_str("GAUGE","VELOCITY", "LENGTH VELOCITY BOTH");
```

Comment:
How about "Specifies the choice of representation of the electric dipole operator. For polarizability, this keyword is ignored and ``LENGTH`` gauge is computed. For optical rotation and raman optical activity, this keyword is active, and acceptable values are ``LENGTH`` for length-gauge representation, ``VELOCITY`` (default) for the modified velocity-gauge representation in which the static-limit tensor is subtracted from the frequency dependent tensor, or ``BOTH``. Note that only velocity gauge results are origin independent."

Is that correct and any clearer?